README.md

```diff
@@ -44,13 +44,13 @@ $(document).ready(function() {
 You can hook to the `play` and `pause` events like this:
 
 ```js
-$(window).on("quicktube:play", function(quicktubeId, $quicktubeEl) {
-    $("[data-show-while-playing]").show();
-});
+window.addEventListener('quicktube:play', () => {
+    showingItem.style.display = 'block';
+}, false);
 
-$(window).on("quicktube:pause", function(quicktubeId, $quicktubeEl) {
-    $("[data-show-while-playing]").hide();
-});
+window.addEventListener('quicktube:pause', () => {
+    showingItem.style.display = 'none';
+}, false);
```

Comment:
IMHO we should completely remove those `quicktube:play` and `quicktube:pause` events. Events make it very hard to understand how the code works – if we want to provide a way for code to run on play/pause, it should be with callbacks.

Now that this supports multiple videos, events make even less sense because they make it harder to know for which video the event was fired.